pandas/tests/indexes/multi/test_missing.py

```diff
@@ -83,7 +83,7 @@ def test_nulls(idx):
         idx.isna()
 
 
-@pytest.mark.xfail
+@pytest.mark.xfail(strict=True)
```

Comment:
can u add a reason

Reply:
Sure, will try to sort that out.  Also waiting to see if anything fails on the CI that passed locally.